Cargo.toml

```diff
@@ -18,6 +18,8 @@ documentation = "https://docs.rs/multihash/"
 edition = "2018"
 
 [dependencies]
+blake2 = { version = "0.7", default-features = false }
```

Comment:
Please use `blake2b_simd` and `blake2s_simd` instead. I know this changes what rust-libp2p does, but they are much better in terms of performance